galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.docker.manager/pom.xml

```diff
@@ -47,7 +47,12 @@
 			<artifactId>commons-io</artifactId>
 			<version>2.6</version>
 		</dependency>
+	
+		<dependency>
```

Comment:
does compress need to be added to docker or http managers?   don't believe it needs to be both.